english/marshmallow.md

```diff
@@ -1,3 +1,6 @@
+This is a story to look for your own, to [fight your way out of this](fight-out/fight-out.md) or to calm down 
+and search for your [destiny](destiny/destiny.md).
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.